crates/libs/windows/src/core/unknown.rs

```diff
@@ -7,6 +7,18 @@ use super::*;
 #[repr(transparent)]
 pub struct IUnknown(core::ptr::NonNull<core::ffi::c_void>);
 
+impl IUnknown {
+    /// Creates an `IUnknown` value by taking ownership of the `raw` pointer.
```

Comment:
It would be good to point out what the `raw` pointer is expected to be (i.e., an owned pointer to a vtable pointer).  You should also note that when the returned value is dopped, the reference count will be decremented. 

It's convention to put the requirements for using this function safely in a section labeled `# Safety`.